tests/main/lxd/task.yaml

```diff
@@ -7,9 +7,7 @@ backends: [-autopkgtest]
 # Only run this on ubuntu 16+, lxd will not work on !ubuntu systems
 # currently nor on ubuntu 14.04
 # ubuntu-18.04-32: i386 is not supported by lxd
-# ubuntu-21.10-*: failing currently with
-# "Setup snap "core" (11993) security profiles (cannot run udev triggers: exit status 1"
-systems: [ubuntu-16*, ubuntu-18.04-64, ubuntu-20*, ubuntu-21.04-*, ubuntu-core-*]
+systems: [ubuntu-16*, ubuntu-18.04-64, ubuntu-20*, ubuntu-21.04-*, ubuntu-21.10-*, ubuntu-core-*]
```

Comment:
nitpick this could become `ubuntu-2*` now couldn't it? I suppose maybe that would cause ubuntu-22.04 to start running sooner than we plan since we will add that image to spread before lxd supports it I suppose